spiffworkflow-frontend/src/components/SpiffTooltip.tsx

```diff
@@ -11,7 +11,7 @@ interface OwnProps {
 
 export default function SpiffTooltip({ title, children }: OwnProps) {
   return (
-    <Tooltip title={title} arrow>
+    <Tooltip title={title} arrow enterDelay={500}>
```

Comment:
unrelated, but just wanted to delay a bit longer than the default of 100 milliseconds before popping open the tooltips.